core/src/io/anuke/mindustry/ui/ContentDisplay.java

```diff
@@ -85,6 +85,9 @@ public static void displayItem(Table table, Item item){
 
         table.left().defaults().fillX();
 
+        table.add(Core.bundle.format("item.corestorable", item.type == ItemType.material ? Core.bundle.format("yes") : Core.bundle.format("no")));
+        table.row();
+
```

Comment:
Unnecessary line break.

Reply:
I feel like "storable" should be separated from explosiveness/flammability/radioactivity because the last tree is "sort of grouped", but I can remove the line if required.

Reply:
I think it's okay with the indent.